### ISSUE TITLE
Invalid antecedent_id

### ISSUE THREAD

**Issue by mlj** [proiel/proiel-treebank#3]
Some tokens (still) reference antecedent tokens that do not exist:

```
armenian-nt.xml is invalid
* Token 737694: antecedent_id references an unknown token
* Token 814655: antecedent_id references an unknown token
* Token 814656: antecedent_id references an unknown token
* Token 1169890: antecedent_id references an unknown token
* Token 822366: antecedent_id references an unknown token
* Token 1079278: antecedent_id references an unknown token
* Token 1079279: antecedent_id references an unknown token
* Token 835116: antecedent_id references an unknown token
* Token 835120: antecedent_id references an unknown token
* Token 838521: antecedent_id references an unknown token
* Token 866518: antecedent_id references an unknown token
* Token 866522: antecedent_id references an unknown token
cic-att.xml is invalid
* Token 1171944: antecedent_id references an unknown token
marianus.xml is invalid
* Token 1161855: antecedent_id references an unknown token
```

These should either be filtered before producing the next release or corrected by reviewers.

**Comment by mlj** [comment]
Most of these invalid references are still in our dev data. This is from the latest run of the release script:

```
armenian-nt.xml is invalid
* Token 737694: antecedent_id references an unknown token
* Token 814655: antecedent_id references an unknown token
* Token 814656: antecedent_id references an unknown token
* Token 1169890: antecedent_id references an unknown token
* Token 822366: antecedent_id references an unknown token
* Token 1079278: antecedent_id references an unknown token
* Token 1079279: antecedent_id references an unknown token
* Token 835116: antecedent_id references an unknown token
* Token 835120: antecedent_id references an unknown token
* Token 838521: antecedent_id references an unknown token
* Token 866518: antecedent_id references an unknown token
* Token 866522: antecedent_id references an unknown token
marianus.xml is invalid
* Token 1161855: antecedent_id references an unknown token
```

This is a release blocker and must be resolved before our next release can be tagged and deposited.

**Comment by mlj** [comment]
Looks like we implemented a fix here

https://github.com/proiel/proiel-cli/commit/479b3190991e03b9ffa27672e96baf7018d48748

but we never released it.

**Comment by mlj** [comment]
I've updated the release scripts to include the patch from proiel-cli.

The invalid IDs have been fixed in release 20180408.
